src/python_tools/tftp_transfer/client.py

```diff
@@ -0,0 +1,23 @@
+import tftpy, argparse
+
+# parse command line arguments
+ap = argparse.ArgumentParser()
+ap.add_argument("-p", "--port", type=int, default=42000, help="inbound port")
```

Comment:
I forget how `ArgumentParser` works but will it tell somewhere the port is `42000` if it isn't passed in? Presumably besides in `-h`.